src/main/java/org/apache/maven/plugins/assembly/io/URLLocation.java

```diff
@@ -60,7 +60,7 @@
     }
 
     /** {@inheritDoc} */
-    void initFile()
+    protected void initFile()
```

Comment:
wouldn't `@Override` annotation be a better choice than `{@inheritDoc}` javadoc

Reply:
`@Override` added